src/apps/api/app.py

```diff
@@ -3,10 +3,12 @@
 from . import endpoints
 from . import api_helpers
 from src.apps import services
+from flask_cors import CORS
 
 # Initialize app
 app = Flask(__name__)
 api = api_helpers.DashApi(app, prefix="/api/v1")
+CORS(app)
```

Comment:
we'ree doing this deliberately because of the need to access the API from other domains like from carbonmark.com right?

It's a mild security issue but common practice for externally accessed APIs

Reply:
Now that I think about it, Only the vercel server is accessing the API for the moment with my latest changes. Let me remove CORS completly.